index.html

```diff
@@ -42,6 +42,61 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
                 </div>
             </div>
         </section>
+        <section id="section-hebergements">
+            <h2>Hébergements à Marseille</h2>
+            <a href="#">Afficher plus</a>
+            <div id="container-herbegement-aticles">
+                <a href="#">
+                    <article>
+                        <img src="images/hebergements/Auberge-de-la-cannebiere.jpg" alt="Auberge de la Cannebière">
+                        <H3>Auberge de la Cannebière</H3>
+                        <p>nuit à partir de <span class="prices">25€</span></p>
+                        <span><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i><span class="grey-stars"><i class="fas fa-star"></i></span></span>
+                    </article>
+                </a>
+                <a href="#">
+                    <article>
+                        <img src="images/hebergements/Hotel-du-port.jpg" alt="">
+                        <h3>Hôtel du port</h3>
+                        <p>Nuit à partir de <span class="prices">52€</span></p>
+                        <span><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i></span>
+                    </article>
+                </a>
+                <a href="#">
+                    <article>
+                        <img src="images/hebergements/Hotel-des-mouettes.jpg" alt="">
+                        <h3>Hôtel des mouettes</h3>
+                        <p>Nuit à partir de <span class="prices">76€</span></p>
+                        <span><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i><span class="grey-stars"><i class="fas fa-star"></i></span></span>
```

Comment:
Prefer to use div not span. Please indent rout code and use always the same star. We ocnfigure active or inactive later.